docs/configuration.md

```diff
@@ -22,28 +22,28 @@ This document provides a comprehensive list of all configurable properties in th
-| config.export.kubeSecret.names           | CONFIG_EXPORT_KUBESECRET_NAMES           | -       | Yes       | config.export.storageType=KUBE_SECRET     | Comma separated names of the Kubernetes Secrets used for DIAL configuration export                                                          |
-| config.export.kubeSecret.key             | CONFIG_EXPORT_KUBESECRET_KEY             | kube-secret-key      | No (recommended to adjust for target environment)      | config.export.storageType=KUBE_SECRET         | Key in Kubernetes Secret used for DIAL configuration export                                                                                 |
-| config.export.keyvault.type              | CONFIG_EXPORT_KEYVAULT_TYPE              | none                 | No (recommended to adjust for target environment)      | -         | Type of keyvault storage for secret values (none, azure, vault, aws, gcp)                                                                   |
-| config.export.keyvault.secretNames       | CONFIG_EXPORT_KEYVAULT_SECRETNAMES       | -                    | Yes  | config.export.keyvault.type in [azure, aws, gcp]         | Names of secrets in keyvault (used when keyvault.type is azure, vault, or aws)                                                              |
-| config.export.keyvault.secretPath        | CONFIG_EXPORT_KEYVAULT_SECRETPATH        | -                    | Yes | config.export.keyvault.type == vault         | Path to secrets in keyvault (used when keyvault.type is vault)                                                                              |
-| config.export.keyvault.expiration.period | CONFIG_EXPORT_KEYVAULT_EXPIRATION_PERIOD | 3                    | No       | -         | Expiration period for keyvault values                                                                                                       |
-| config.export.keyvault.expiration.unit   | CONFIG_EXPORT_KEYVAULT_EXPIRATION_UNIT   | MONTHS               | No       | -         | Unit of time for keyvault value expiration                                                                                                  |
-| config.export.createResources            | CONFIG_EXPORT_CREATE_RESOURCES           | false                | No       | -         | If true, create resources where config is exported if they don't already exist                                                              |
+| Setting                                     | Environment Variable                      | Default              | Required                                                                                              | Applied when                                     | Description                                                                                                                                 |
```

Comment:
it's only formatting? what reason?

Reply:
let's revert. you can propose formatting changes in specialized PR. (it's changes not related to PR goal)